frontend/src/pages/roles/components/ViewPermissions.tsx

```diff
@@ -0,0 +1,92 @@
+import Switch from '@mui/material/Switch';
+import RoleDTO from '@shared/dtos/RoleDTO';
+import { Permission } from '@shared/utils/Permission';
+import { useState } from 'react';
+
+const PermissionsLists: { roles: Permission[]; users: Permission[]; meetings: Permission[] } = {
+    roles: ['VIEW_ROLES', 'MANAGE_ROLES'],
+    users: ['VIEW_USERS', 'MANAGE_USERS'],
+    meetings: ['VIEW_MEETINGS', 'MANAGE_MEETINGS'],
+};
+
+export function ViewPermissions({
+    role,
+    savePermissions,
+}: {
+    role: RoleDTO;
+    savePermissions: (role: RoleDTO) => void;
+}) {
+    function checkSwitch(perm: Permission) {
+        if (['Admin', 'Default'].indexOf(role.name) !== -1) {
```

Comment:
Change this to `.includes` ? Let's talk about this

Reply:
Done